devicetypes/smartthings/zwave-siren.src/zwave-siren.groovy

```diff
@@ -126,10 +160,37 @@ def configure() {
 		cmds << "delay 500"
 		// Set alarm sound to no.10
 		cmds << zwave.configurationV1.configurationSet(parameterNumber: 5, size: 1, configurationValue: [10]).format()
+	} else if (isYale()) {
+		if (!state.alarmLength) state.alarmLength = 10 // default value
+		if (!state.alarmLEDflash) state.alarmLEDflash = 1 // default value
+		if (!state.comfortLED) state.comfortLED = 0 // default value
+		if (!state.tamper) state.tamper = 1 // default value
+
+		log.debug "settings: ${settings.inspect()}"
+		log.debug "state: ${state.inspect()}"
+
+		Short alarmLength = (settings.alarmLength as Short) ?: 10
+		Boolean alarmLEDflash = (settings.alarmLEDflash as Boolean) == null ? true : settings.alarmLEDflash
+		Short comfortLED = (settings.comfortLED as Short) ?: 0
+		Boolean tamper = (settings.tamper as Boolean) == null ? true : settings.tamper
+
+		if (alarmLength != state.alarmLength || alarmLEDflash != state.alarmLEDflash || comfortLED != state.comfortLED || tamper != state.tamper) {
+			state.alarmLength = alarmLength
+			state.alarmLEDflash = alarmLEDflash
+			state.comfortLED = comfortLED
+			state.tamper = tamper
+
+			cmds << zwave.configurationV1.configurationSet(parameterNumber: 1, size: 1, configurationValue: [alarmLength]).format()
+			cmds << zwave.configurationV1.configurationSet(parameterNumber: 2, size: 1, configurationValue: [alarmLEDflash ? 1 : 0]).format()
+			cmds << zwave.configurationV1.configurationSet(parameterNumber: 3, size: 1, configurationValue: [comfortLED]).format()
+			cmds << zwave.configurationV1.configurationSet(parameterNumber: 4, size: 1, configurationValue: [tamper ? 1 : 0]).format()
```

Comment:
For parallel structure's sake, you should call `format()` on all the commands or none of them.